src/views/registration-js/mini-cart.php

```diff
@@ -49,30 +47,35 @@
 	<h3 class="tribe-common-h6 tribe-common-h5--min-medium tribe-common-h--alt tribe-tickets__mini-cart__title"><?php echo esc_html_x( 'Ticket Summary', 'Attendee registration mini-cart/ticket summary title.', 'event-tickets'); ?></h3>
 		<?php foreach ( $events as $event_id => $tickets ) : ?>
 			<?php foreach ( $tickets as $key => $ticket ) : ?>
-				<?php if ( $provider !== $ticket['provider']->attendee_object ) : ?>
+				<?php if ( $cart_provider !== $ticket['provider']->attendee_object ) : ?>
 					<?php continue; ?>
 				<?php endif; ?>
-				<?php $currency_symbol     = $currency->get_currency_symbol( $ticket['id'], true ); ?>
+				<?php $currency_symbol = $currency->get_currency_symbol( $ticket['id'], true ); ?>
 				<?php $this->template(
 					'blocks/tickets/item',
 					[
-						'ticket' => $provider_obj->get_ticket( $event_id, $ticket['id'] ),
-						'key' => $key,
-						'is_mini' => true,
+						'ticket'          => $cart_provider->get_ticket( $event_id, $ticket['id'] ),
+						'key'             => $key,
+						'is_mini'         => true,
 						'currency_symbol' => $currency_symbol,
-						'provider' => $provider_obj,
-						'post_id' => $event_id
-						]
+						'provider'        => $cart_provider,
+						'post_id'         => $event_id,
+					]
 				); ?>
 			<?php endforeach; ?>
 		<?php endforeach; ?>
-		<?php $this->template( 'blocks/tickets/footer', [ 'is_mini' => true, 'provider' => $provider_obj ] ); ?>
+	<?php $this->template( 'blocks/tickets/footer', [ 'is_mini' => true, 'provider' => $cart_provider ] ); ?>
```

Comment:
:no_entry_sign: When a multi-item array uses associative keys, each value should start on a new line.

<sub>hash: 50a88f776c24d42486906b2cec10fdce</sub>